### DISCUSSION TITLE
Start menu - recommended

### DISCUSSION THREAD

**Discussion by synapse46** [LeDragoX/Win-Debloat-Tools#176] [General]
Is there a setting that is disabling the Recommended section on the start menu.  I've found on the home version it can't be removed so might as well make it usable.  Thanks for the great script!

**Comment by LeDragoX** [comment]
I didn't researched in that area, but if I have the time, I'm willing to make the Start Menu more clean (in a non-destructive way), I know the pinned apps are managed by a JSON file somewhere, but I'll need to do more research and _may_ come in a future update™.

Thanks for the compliment <3
